README.md

```diff
@@ -64,7 +71,9 @@ Just remember that You need to `activate Polymake.jl` to use `Polymake`.
 
 ## Examples
 
-`polymake` big objects (like `Polytope`, `Cone`, etc) can be created with the `@pm` macro:
+In this section we just highlight various possible uses of `julia/polymake`. Please refer to [Polymake syntax translation](#polymake-syntax-translation) for more thorough treatment.
```

Comment:
I like `Polymake.jl` more than `julia/polymake`. And `polymake` instead of `perl/polymake`.